### PR TITLE
Danilo/csb 517 issues template

### PR DESCRIPTION
**Pull request by danilowoz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: 🐛 Bug Report
+about: Did something not work?
```
Comment (JamesACS) [inline .github/ISSUE_TEMPLATE/BUG.md:3]: Perhaps 'What went wrong?' instead?

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# 🐛 bug report
+
+## Packages affected
+
+- [ ] Sandpack-client
+- [ ] Sandpack-react
+
+## Description of the problem
+
+## How has this issue affected you? What are you trying to accomplish?
```
Comment (JamesACS) [inline .github/ISSUE_TEMPLATE/BUG.md:17]: The back to back questions here come across as almost passive aggressive? Maybe rephrase:

What were you doing when the problem occurred'?

File: .github/ISSUE_TEMPLATE/DOCUMENTATION.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+name: 📃 Documentation
+about: Report an issue related to documentation
+title: 'Documentation: '
+labels: 'documentation'
+---
```
Comment (JamesACS) [inline .github/ISSUE_TEMPLATE/DOCUMENTATION.md:6]: Will further fields be added here? Similar to the bug report? With documentation we may want various categories for genuine technical errors vs say, grammar errors

File: .github/ISSUE_TEMPLATE/FEATURE.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+## Packages affected
+
+- [ ] Sandpack-client
+- [ ] Sandpack-react
+
+## What is this feature?
+
+## How the feature should work?
+
+## You have examples or an idea how it can be implemented?
```
Comment (JamesACS) [inline .github/ISSUE_TEMPLATE/FEATURE.md:19]: Making it a bit more friendly: Do you have any examples of how you would like to see us implement it?

File: .github/PULL_REQUEST_TEMPLATE.md.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Please make sure you are familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+## What kind of change does this PR introduce?
+
+<!-- Is it a Bug fix, feature, docs update, ... -->
```
Comment (JamesACS) [inline .github/PULL_REQUEST_TEMPLATE.md.md:3]: I'd lean away from using abbreviations, even commonly accepted ones like PR - as these forms are in English people may use translation software if they're not fluent, so making sure to just say documents or pull requests rather than docs or PR

File: .github/ISSUE_TEMPLATE/DOCUMENTATION.md (hunk 3 above)
Comment (danilowoz) [reply .github/ISSUE_TEMPLATE/DOCUMENTATION.md:6]: @JamesACS do you have something in mind? I tried to find some examples, but I didn't find anything that I liked
